src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -119,7 +119,7 @@ class RoutedSavableReviewPage extends React.Component {
             <strong>We’re sorry. We can't submit your form right now.</strong>
           </p>
           <p>
-            We're working to fix the problem. Please make sure you're connected
+            We’re working to fix the problem. Please make sure you’re connected
```

Comment:
Oh look, I get to keep my apostrophe changes. This was initially part of the wider ticket to update content for wbc but the other changes got dropped because this happened in another ticket (per comments).